pkgs/development/python-modules/aiopegelonline/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, aiohttp
+, aioresponses
+, buildPythonPackage
+, fetchFromGitHub
+, pytest-asyncio
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "aiopegelonline";
+  version = "0.0.6";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "mib1185";
+    repo = "aiopegelonline";
+    rev = "refs/tagsv${version}";
```

Comment:
I think there's a typo here that the FOD being cached might have caused you to miss locally.

Reply:
Thanks for checking, updated.